### PR TITLE
PowerToys: Add two examples of RegEx search/replace

### PR DESCRIPTION
**Pull request by Jay-o-Way**
Small one.
Adressing https://github.com/microsoft/PowerToys/issues/685 and https://github.com/microsoft/PowerToys/issues/1585

### CODE REVIEW

**Comment by PRMerger4** [comment]
@Jay-o-Way : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Jak-MS** [comment]
@mattwojo 

Can you review this PR?

IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by Jay-o-Way** [comment]
@Jak-MS Interesting that you do the same thing as what the bot already did 🙃

File: hub/powertoys/powerrename.md [hunk 1]
```diff
@@ -151,28 +147,30 @@ To use the [Boost library](https://www.boost.org/doc/libs/1_74_0/libs/regex/doc/
...
 
-| Search for                 | Replace with | Description |
-| :---                       | :---         | :--- |
-| `(.*).png`                 | `foo_$1.png` | Prepends "foo\_" to the existing file name for PNG files |
-| `(.*).png`                 | `$1_foo.png` | Appends "\_foo" to the existing file name for PNG files |
-| `(.*)`                     | `$1.txt`     | Appends ".txt" extension to existing file |
-| `(^\w+\.$)¦(^\w+$)`        | `$2.txt`     | Appends ".txt" extension to existing file name only if it does not have an extension |
-| `(\d\d)-(\d\d)-(\d\d\d\d)` | `$3-$2-$1`   | Move numbers in the filename: "29-03-2020" becomes "2020-03-29" |
+| Search for                                              | Replace with | Description                                                                          |
+|:--------------------------------------------------------|:-------------|:-------------------------------------------------------------------------------------|
```
Comment (mattwojo) [inline hub/powertoys/powerrename.md:166]: Just a note about this table formatting: The dashes here are just a formatting preference for the markdown. Doesn't impact the way the table appears after being built.

**Review by mattwojo — APPROVED** [review]
Looks good

**Comment by mattwojo** [comment]
#sign-off

File: hub/powertoys/powerrename.md (hunk 1 above)
Comment (Jay-o-Way) [reply hub/powertoys/powerrename.md:166]: Hah I know, just clicked "expand this table" in VS Code. I can use both.

Comment (mattwojo) [reply hub/powertoys/powerrename.md:166]: Oh! Right on - haven't tried that. Good to know that's how it handles it. I've seen writers have strong opinions on what is "best" (which clearly doesn't matter... but you know, folks love to have strong opinions). 😜

**Pull request merged**